From 451fa0d4c36871f7ef302e8dd908694016b8fa4e Mon Sep 17 00:00:00 2001
From: Jean Delu <jeandelu@tutamail.com>
Date: Sat, 8 Aug 2026 13:56:32 +0200
Subject: [PATCH 12369/15333] parisc: eisa_eeprom: Add missing
 MODULE_DESCRIPTION()

drivers/parisc/eisa_eeprom.c is a standalone module (it has MODULE_LICENSE()
and module_init()) but is missing the MODULE_DESCRIPTION() macro, so modpost
emits a warning about it.  Add the missing description.

Signed-off-by: Jean Delu <jeandelu@tutamail.com>
Signed-off-by: Helge Deller <deller@gmx.de>
---
 drivers/parisc/eisa_eeprom.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/parisc/eisa_eeprom.c b/drivers/parisc/eisa_eeprom.c
index 601cbb22574f..2058b4c5babe 100644
--- a/drivers/parisc/eisa_eeprom.c
+++ b/drivers/parisc/eisa_eeprom.c
@@ -94,6 +94,7 @@ static int __init eisa_eeprom_init(void)
 	return 0;
 }
 
+MODULE_DESCRIPTION("EISA eeprom driver");
 MODULE_LICENSE("GPL");
 
 module_init(eisa_eeprom_init);
-- 
2.55.0

